script/utils.js

```diff
@@ -7,6 +7,12 @@ const runCommand = (cmd, forcedExitCode = null) => {
   });
 };
 
+const runCommandSync = (cmd, forcedExitCode = null) => {
+  const child = spawnSync(cmd, [], { shell: true, stdio: 'inherit' });
```

Comment:
Leaving this here since I googled it for myself

`spawnSync` docs: https://nodejs.org/api/child_process.html#child_process_child_process_spawnsync_command_args_options